ngraph/src/ngraph/pass/graph_rewrite.hpp

```diff
@@ -82,6 +110,33 @@ class NGRAPH_API ngraph::pass::GraphRewriteBase : public ngraph::pass::FunctionP
 class NGRAPH_API ngraph::pass::GraphRewrite : public ngraph::pass::GraphRewriteBase
 {
 public:
+    GraphRewrite()
+        : GraphRewriteBase()
```

Comment:
Again calling base class default c-tor for... trivial c-tor. Recommend to replace by `= default`.

Reply:
done.